crates/turbopack-dev-server/src/source/asset_graph.rs

```diff
@@ -150,7 +150,10 @@ async fn expand(
     Ok(map)
 }
 
-// This must not be a TaskInput since this would resolve the embedded asset
+/// A unresolve asset. We need to have a unresolve Asset here as we need to
+/// lookup the Vc identity in the expanded set.
```

Comment:
Sorry, I still don't understand. What is it about a resolved asset that prevents vc identity lookup? How does an unresolved asset solve that?